src/controllers/userController.js

```diff
@@ -1,65 +1,92 @@
-const userSchema = require('../models/userSchema')
+const userSchema = require('../models/userSchema');
 const mongoose = require('mongoose');
 
 /**
  * Return all users
- * @param {*} res 
- * @param {*} req 
+ * @param {*} res
+ * @param {*} req
  */
-async function getAllUsers (req, res) {
-    
-    //  Implement Your Code Here
-
+async function getAllUsers(req, res) {
+  try {
+    const users = await userSchema.find();
+    res.json(users);
+  } catch (error) {
+    res.status(500).json({ error: error });
+  }
 }
 
 /**
  * Sig UP the user
- * @param {*} req 
- * @param {*} res 
+ * @param {*} req
+ * @param {*} res
  */
-async function signUp (req, res){
-
-    //  Implement Your Code Here
-
+async function signUp(req, res) {
+  try {
+    const { email, password, role } = req.body;
```

Comment:
before saving in the database please check this is values email password role are null, undefined or '' in inputs. otherwise you will error instead shows a error which input are null empty.